src/registrar/tests/test_admin.py

```diff
@@ -1667,91 +1653,6 @@ def test_readonly_when_restricted_superuser(self):
 
             self.assertEqual(readonly_fields, expected_fields)
 
-    def test_change_view_for_joined_contact_five_or_less(self):
```

Comment:
Delete tests for joined contacts since we no longer use Submitter to join Contacts and Domains/Domain Requests